garble.py

```diff
@@ -78,10 +79,13 @@ def garble_pii(args):
         source_file = Path(args.sourcefile)
     else:
         filenames = list(
-            filter(lambda x: "pii" in x and len(x) == 23, os.listdir("temp-data"))
+            filter(
+                lambda x: "pii" in x and len(x) == 8 + TIMESTAMP_LEN,
+                os.listdir("temp-data"),
+            )
         )
         timestamps = [
-            datetime.strptime(filename[4:-4], "%Y%m%dT%H%M%S") for filename in filenames
+            datetime.strptime(filename[4:-4], TIMESTAMP_FMT) for filename in filenames
```

Comment:
Copying over this note from the corresponding PR on linkage-agent-tools. I won't hold up the merge just for this, but maybe there's an easy fix I haven't thought of:

Looks like we would need to be aware of this line if we do change `TIMESTAMP_FMT` -- the length of the snippet being parsed out will depend on the date format. Can you think of a clean way we can handle that here? Unfortunately the length of a timestamp will be different than the length of the format string (for example `len("2022-11-10") != len("%Y-%m-%d")` )

Reply:
Oh the answer is right there -- the new constant `TIMESTAMP_LEN`